<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToGrayOptionWidget</class>
 <widget class="QWidget" name="ToGrayOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>113</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="greenButton">
     <property name="toolTip">
      <string>g
Worst effect, but fastest.</string>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="averageButton">
     <property name="toolTip">
      <string>(b + g + r) / 3
Poor effect, but fast.</string>
     </property>
     <property name="text">
      <string>Average</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="floatButton">
     <property name="toolTip">
      <string>0.11 * b + 0.59 * g + 0.3 * r
Best effect, but slowest.</string>
     </property>
     <property name="text">
      <string>Float</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="integerButton">
     <property name="toolTip">
      <string>(11 * b + 59 * g + 30 * r) / 100
Similar effect to Float, a little faster.</string>
     </property>
     <property name="text">
      <string>Integer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="displacementButton">
     <property name="toolTip">
      <string>(77 * b + 151 * g + 28 * r) &gt;&gt; 8
Similar effect to Integer, a little faster.</string>
     </property>
     <property name="text">
      <string>Displacement</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
